### PR TITLE
Add a method to get Prismic oauth authentication URL

### PR DESCRIPTION
**Pull request by dlecan**
This method allows you to get the URL where to authenticate your Prismic user to get a OAuth token.

### CODE REVIEW

File: src/angularPrismicio.js [hunk 1]
```diff
@@ -291,6 +324,7 @@ angular.module('prismic.io', [])
         Configurer.init(service, config);
         service.api = api;
         service.ctx = ctx;
+        service.getAuthenticationUrl = getAuthenticationUrl;
```
Comment (awulder) [inline src/angularPrismicio.js]: None of the properties are prefixed with get* or set*
I would suggest to rename it to authenticationUrl.

**Comment by awulder** [comment]
Nice, thank you. I have added one comment and after that I will merge your PR.

File: src/angularPrismicio.js (hunk 1 above)
Comment (dlecan) [inline src/angularPrismicio.js]: You're right, much better without "get" :) I will update the PR

**Comment by awulder** [comment]
Thanks, could you please sync this branch with master so I can merge your pull request?

**Comment by dlecan** [comment]
> could you please sync this branch with master so I can merge your pull request?

Done

**Pull request merged**